### ISSUE TITLE
UI todos

### ISSUE THREAD

**Issue by rebeccacremona** [harvard-lil/perma-capture#76]
Stream of consciousness:

- ~we have to include a link to the Harvard accessibility policy~
- ~the log out link has no href~
- ~there is a "logged out" template, but the page doesn't log you out of Django yet~
- ~the chevron to toggle playback is not accessible by keyboard: switch it to a button element~ Probably a designer would not like the focus CSS now; I note there is no hover style. Address when doing more keyboard nav work.
- handle keyboard focus, when you open the column to show details. right now, you have to tab through the whole capture list to get there. hmmm. this is going to be a little tricky....... maybe I'll do it like a modal......
- ~playback is going to the `search://view=pages` instead of the actual playback~
- ~this `active` calculation is not serving as desired~
- start saving favicon and page title and displaying instead of placeholders, if available
- restore the ability to sort and pull up a particular label
- ~login (or generic) form validation error text spacing~
- why, in Firefox, when following validation error links, are focus styles not applied? pressing tab proves keyboard focus IS moving. works fine in chrome and safari.... 
- ~white padding at the bottom on mobile on purpose or by accident?~
- ~red x for failed capture might need clarification text; this kind of looks like a delete button~
- ~maybe switch "Recorded [date]" to "Captured [date]"? what language is most consistent?~
- ~see if adjusting font sizes at some medium screen sizes helps, or consider breaking a little earlier~
- indication that account edit form has submitted successfully
- reset password page styled to match. and, after you hit the button, needs success indication
- add back the a11y feature that announces your api key has successfully been changed; maybe integrate that with whatever flash message says your email/name changes worked too?
- password change success message, and reset form.
- ~the top of the docs page needs more space~
- to be discussed: is docs TOC hierarchy sufficiently visually distinct?
- take a pass through everything with VoiceOver
- Re-enable sign up, and build out those pages, for the time when we are ready
- consider adding behavior for logged out top nav on mobile, which wraps if zoomed... but I think only if zoomed

  ![image](https://user-images.githubusercontent.com/11020492/128231967-5a73af96-0495-4b23-97a6-0b47e5a59490.png)

... And obviously I have to rewrite the docs, so that they describe the new app!!!

**Comment by bensteinberg** [comment]
When you have a long list of captures (not expiring?) and you click the chevron for playback, the playback is way back at the top of the right-hand panel, not where you are scrolled to lower down in the page.

**Comment by rebeccacremona** [comment]
add an input for headless/headful, and for login_if_supported, maybe in a collapsed options section

**Comment by rebeccacremona** [comment]
add browser info and logged in bool below capture size in details (for now at least; if people think irrelevant/cluttered np)

**Comment by rebeccacremona** [comment]
~"recorded at" and "expires at" both show creation time~

**Comment by rebeccacremona** [comment]
~Restore lazy loading of capture list~

**Comment by rebeccacremona** [comment]
~Invalid capture jobs should show their error message, instead of looking just like failed jobs. Probably, `submitted_url` should not be hyperlinked.~ Think about this for mobile view, though, where `title` is not presently shown.

**Comment by rebeccacremona** [comment]
Map API error responses to human-friendly error messages for display

**Comment by rebeccacremona** [comment]
~submitted URL should only be clickable on link text, not full whitespace of grid element~

**Comment by rebeccacremona** [comment]
Fields "labeled" visually via placeholder all have their labels disappear when you tab in: switch to normal labels, or the fancy kind like CAP has.

**Comment by rebeccacremona** [comment]
look into CNN preview. is it wonky because the preview pane is narrow / we are capturing at large width?

**Comment by rebeccacremona** [comment]
~the date formatting util wants to be a filter https://v1.vuejs.org/guide/custom-filter.html~ (filters are not part of Vue3)

**Comment by rebeccacremona** [comment]
fix method duplication in ItemDetail and ListItem

**Comment by rebeccacremona** [comment]
> When you have a long list of captures (not expiring?) and you click the chevron for playback, the playback is way back at the top of the right-hand panel, not where you are scrolled to lower down in the page.

Fixed (except for keyboard navigators) by https://github.com/harvard-lil/perma-capture/pull/80

**Comment by rebeccacremona** [comment]
~Oh, now the capture details panel, in 3-column code, also needs its height restricted.~

**Comment by rebeccacremona** [comment]
On resizing: maybe I can set a watcher on the window height var instead of listening for the resize event: the value in the store is updated by a resize listener on the dashboard

**Comment by rebeccacremona** [comment]
Page titles are absent/wonky. Might need to check in about the research project there: there was some concern re: google and indexing if Vue handled the titles.

**Comment by rebeccacremona** [comment]
404/500/other error pages

**Comment by rebeccacremona** [comment]
mockups show:
- perma-like footer
- special display if no captures
- download all button
- "expires in x minutes"

also, consider changing mobile tray to a gray after expiration (tan looks nice against the white but weird against the gray)

**Comment by rebeccacremona** [comment]
Wireframes link, which I keep losing: https://www.figma.com/file/yDSQHz4QtwSFxWJhyQ08JT/Perma-capture?node-id=202%3A297

**Comment by rebeccacremona** [comment]
The proportions of the grid are a little off. 
- The footer is a little too low: you can't visually tell it's there. 
- In two-column mode, the content is noticeably pulled to the sides of the screen with a wide gutter in the center. Compare to the wireframes, where the content comfortably fills the screen.

![image](https://user-images.githubusercontent.com/11020492/134957470-1e0bd000-a01d-488f-a97a-c65c9dcf39cb.png)
![image](https://user-images.githubusercontent.com/11020492/134957675-e3821e39-8cb1-4535-90b2-8208b5fe33e6.png)

**Comment by rebeccacremona** [comment]
Footer on docs page looks kind of weird against the gray of the content-centering padding. Situation not covered by wireframes.

**Comment by rebeccacremona** [comment]
Copy and application name need another pass.

**Comment by rebeccacremona** [comment]
Labels and spacing, and location of warning 
![image](https://user-images.githubusercontent.com/11020492/134958887-c180ebdd-8891-42f5-9953-bc0e64ba9b34.png)

(edit: ah, I think these were supposed to be fancy inline labels too, might take care of it)

**Comment by rebeccacremona** [comment]
Favicon (presently a placeholder img) should be vertically centered with title (also a placeholder)
![image](https://user-images.githubusercontent.com/11020492/134974213-68cb705f-ea9e-4fc4-a86e-dcef169feaa8.png)

**Comment by rebeccacremona** [comment]
Mobile link list: URL should be vertically centered.
![image](https://user-images.githubusercontent.com/11020492/134974785-e671fe16-fbad-4232-bbda-8f2760e252c5.png)

**Comment by rebeccacremona** [comment]
It's weird that the dropdown lets you navigate to the page you are on, with no indication that you are there, I think. Weird, right? Not just me in this moment?

**Comment by rebeccacremona** [comment]
Consider adding progress bars during capture (maybe)

**Comment by rebeccacremona** [comment]
a "no js" fallback message, probably
